src/core/pointcloud/qgspointcloudrequest.h

```diff
@@ -48,8 +50,20 @@ class CORE_EXPORT QgsPointCloudRequest
     //! Set attributes filter in the request
     void setAttributes( const QgsPointCloudAttributeCollection &attributes );
 
+    /**
+     * Returns the rectangle from which points will be taken. If the returned rectangle is empty, then no filter rectangle is set.
+     * \since QGIS 3.30
+     */
+    QgsRectangle filterRect() const { return mExtent; }
```

Comment:
Can you make a note here that the rect must be in the layer's crs?

Reply:
done